README.md

```diff
@@ -171,10 +178,10 @@ Usage of graphite-ch-optimizer:
   -c, --config string                Filename of the custom config. CLI arguments override it
       --print-defaults               Print default config values and exit
   -v, --version                      Print version and exit
-      --optimize-interval duration   The active partitions won't be optimized more than once per this interval, seconds (default 72h0m0s)
+      --optimize-interval duration   The partition will be merged unless it have active writes for more than, duration (default 72h0m0s)
```

Comment:
`The partition will be merged after having no writes for more than the given duration`